ambry-store/src/main/java/com.github.ambry.store/CompactionManager.java

```diff
@@ -140,6 +140,19 @@ boolean scheduleNextForCompaction(BlobStore store) {
     return compactionExecutor != null && compactionExecutor.scheduleNextForCompaction(store);
   }
 
+  /**
+   * Disable the given {@code store} for compaction next.
+   * @param store the {@link BlobStore} to compact.
```

Comment:
The comment description is incorrect. Change "store the {@link BlobStore} to compact" to "store the {@link BlobStore} on which the compaction is disabled".